content/en/docs/getting-started/_index.md

```diff
@@ -60,73 +60,162 @@ olm-operator       1/1     1            1           5m52s
 packageserver      2/2     2            2           5m43s
 ```
 
-## Installing an Operator using OLM
+## Installing OLM with Operator SDK
 
-When you install OLM, it comes packaged with a number of Operators developed by the community that you can install instantly.
-You can use the `pacakagemanifest` api to see the operators available for you to install in your cluster:
+The [`operator-sdk`][operator-sdk] provides a command to easily install and uninstall OLM for developmental purposes. See the [SDK installation guide][sdk-installation-guide] on how to install `operator-sdk` tooling.
 
-```sh
-$ kubectl get packagemanifest -n olm
-NAME                               CATALOG               AGE
-cassandra-operator                 Community Operators   26m
-etcd                               Community Operators   26m
-postgres-operator                  Community Operators   26m
-prometheus                         Community Operators   26m
-wildfly                            Community Operators   26m
+With `operator-sdk` installed, you can easily install OLM on your cluster by running `operator-sdk olm install`. It is just as easy to uninstall OLM by running `operator-sdk uninstall olm`.  For more information about how to integrate your project using [`operator-sdk`][operator-sdk] CLI tool, see the following [OLM integration][sdk-olm-integration] section.
+
+## Installing an operator bundle with Operator SDK
+
+You can use the [`operator-sdk`][operator-sdk] CLI to run your bundle with [`operator-sdk run bundle`][cli-run-bundle].
+
+Now that the bundle image is present in a registry, [`operator-sdk run bundle`][cli-run-bundle]
```

Comment:
`Now that the bundle image is present in a registry` reads like we've just done that step in a previous section.
But that's not the case right? So maybe rephrasing that to make it more like a pre-requisite to have an existing bundle image on some registry. 
```suggestion
Given a bundle image is present in a registry, [`operator-sdk run bundle`][cli-run-bundle]
```

Reply:
Done.